src/showcase/showcase-directive.js

```diff
@@ -11,23 +11,49 @@
                 },
                 link: (scope, element) => {
                     var chart = bitflux.app().quandlApiKey(quandlService.apiKey()),
-                        firstRun = true;
+                        firstRun = true,
+                        store;
+
+                    if (window.storeService) {
+                        store = window.storeService.open(window.name);
+                        chart.indicators(store.indicators());
+                    }
 
                     chart.periodsOfDataToFetch(configService.getBitfluxStockAmount());
                     chart.proportionOfDataToDisplayByDefault(configService.getInitialBitfluxProportion());
 
                     scope.$watch('selection()', (newSelection, previousSelection) => {
                         if (newSelection !== '') {
+                            // The store service may not have been passed through yet.
+                            // If it hasn't been, get it for later, where we set the
+                            // indicators
+                            if (!store && window.storeService) {
+                                store = window.storeService.open(window.name);
+                            }
+
                             if (firstRun) {
                                 firstRun = false;
                                 chart.run(element[0].children[0]);
                             }
 
+                            if (store) {
+                                chart.indicators(store.indicators());
+                            }
+
                             if (newSelection !== previousSelection) {
                                 chart.changeQuandlProduct(newSelection);
                             }
                         }
                     });
+
+                    scope.$watchCollection(() => store.indicators(), (newValues) => {
+                        if (!store && window.storeService) {
+                            store = window.storeService.open(window.name);
+                        }
```

Comment:
Given this is used about 3 times now perhaps it could be a function?

Across the app it's used quite a lot too.. we should probably do something about that.